util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Extensions.cs

```diff
@@ -123,17 +123,73 @@ public ProjectResolver (CachedMavenRepository repository)
 
 	public Project Resolve (Artifact artifact)
 	{
-		if (repository.TryGetFilePath (artifact, $"{artifact.Id}-{artifact.Version}.pom", out var path)) {
-			using (var stream = File.OpenRead (path)) {
-				var pom = Project.Load (stream) ?? throw new InvalidOperationException ($"Could not deserialize POM for {artifact}");
+		try {
+			if (repository.TryGetFilePath (artifact, $"{artifact.Id}-{artifact.Version}.pom", out var path)) {
+				using (var stream = File.OpenRead (path)) {
+					var pom = Project.Load (stream) ?? throw new InvalidOperationException ($"Could not deserialize POM for {artifact}");
 
-				// Use index instead of Add to handle duplicates
-				ResolvedPoms [artifact.VersionedArtifactString] = path;
+					// Use index instead of Add to handle duplicates
+					ResolvedPoms [artifact.VersionedArtifactString] = path;
 
-				return pom;
+					return pom;
+				}
 			}
+		} catch (Exception ex) when (IsHttpError(ex)) {
+			// Handle HTTP errors gracefully by logging and returning a minimal POM
+			Console.WriteLine ($"Warning: Could not resolve POM for {artifact.VersionedArtifactString} (HTTP error: {GetHttpErrorMessage(ex)}). Continuing with minimal dependency information.");
+			return CreateMinimalProject (artifact);
 		}
```

Comment:
Instead of trying to "fix" this...
* Undo all the changes in this method
* Add a `try`-`catch` with a `throw`
* Log the dependency that failed

So instead of just:
```
Dependency errors : 1
1
	System.Net.Http.HttpRequestException: Response status code does not indicate success: 404 (Not Found).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at System.Net.Http.HttpClient.GetStreamAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
```

Let's make it print out as much information about the dependency: id, name, version, etc.